scripts/zones/Grand_Palace_of_HuXzoi/mobs/Ixaern_MNK.lua

```diff
@@ -8,53 +8,94 @@ require("scripts/globals/status")
 -----------------------------------
 local entity = {}
 
+local bracerMode = function(mob, qnAern1, qnAern2)
+    local mobID = mob:getID()
+    mob:useMobAbility(690) -- Hundred Fists
+    if qnAern1:isAlive() then qnAern1:useMobAbility(692) end -- Chainspell
+    if qnAern2:isAlive() then qnAern2:useMobAbility(689) end -- Benediction
+    mob:addMod(xi.mod.ATT, 200)
+
+    for i = mobID+1, mobID+2 do
+        local pet = GetMobByID(i)
+        if pet:isSpawned() then
+            pet:AnimationSub(2)
+            pet:addMod(xi.mod.ATT, 200)
+            pet:setMod(xi.mod.DELAY, 2100)
+        end
+    end
+    -- slightly delay adding local var to avoid adding bracers to Ix'Mnk while Hundred Fists is still active.
+    mob:timer(3000, function(mobArg)
+        mobArg:setLocalVar("enableBracers", 1)
+    end)
+end
+
 entity.onMobSpawn = function(mob)
     -- adjust drops based on number of HQ Aern Organs traded to QM
     local qm = GetNPCByID(ID.npc.QM_IXAERN_MNK)
     local chance = qm:getLocalVar("[SEA]IxAern_DropRate")
-    if (math.random(0, 1) > 0) then
-        SetDropRate(4398, 1851, chance * 10) -- Deed Of Placidity
-        SetDropRate(4398, 1901, 0)
+    if math.random(0, 3) > 0 then
+        SetDropRate(2845, 1851, chance * 10) -- Deed Of Placidity
+        SetDropRate(2845, 1901, 0)
     else
-        SetDropRate(4398, 1851, 0)
-        SetDropRate(4398, 1901, chance * 10) -- Vice of Antipathy
+        SetDropRate(2845, 1851, 0)
+        SetDropRate(2845, 1901, chance * 10) -- Vice of Antipathy
```

Comment:
@zach2good is this still the pattern we want for drop lists that change mid fight? I know we have several ways that work but I no longer remember what the gold standard was.

Reply:
Would like to see xi.items used here as well if this is the route we're going

Reply:
Just changing it so it works, let me know whatever style you want. The droplist it was referencing was incorrect.